fastkafka_gen/_code_generator/asyncapi_spec_generator.py

```diff
@@ -53,6 +53,9 @@ def _validate_response(response: str) -> List[str]:
     Raises:
         json.JSONDecodeError: If the response is not a valid JSON.
     """
+    if "INCOMPLETE APP DESCRIPTION" in response.upper():
```

Comment:
Can we change this line to:

if "==== INCOMPLETE APP DESCRIPTION ====" in response: